hopsworks-admin/src/main/webapp/security/protected/admin/admintitleBar.xhtml

```diff
@@ -50,7 +50,8 @@
       <h:form id="toolbarForm"> 
         <p:toolbar>
           <f:facet name="left" rendered="#{roleEnforcementPoint.loggedIn}">
-            <p:commandButton value="Hopsworks" action="#{facesContext.externalContext.redirect('/hopsworks')}" ajax="false" process="@toolbarForm" icon="ui-icon-extlink"/>
+            <p:commandButton value="Aegis" action="#{facesContext.externalContext.redirect('/aegis-web')}" ajax="false" process="@toolbarForm" icon="ui-icon-extlink"/>
+            <!--<p:commandButton value="Hopsworks" action="#{facesContext.externalContext.redirect('/aegis-web')}" ajax="false" process="@toolbarForm" icon="ui-icon-extlink"/>-->
```

Comment:
same here but since it's a comment could be deleted.

Reply:
done